### PR TITLE
Update outdated packages

### PR DESCRIPTION
**Pull request by michplunkett**
## Describe your changes
I am updating all out-of-date node packages. I also re-ran `gulp` and updated the files in the `docs` folder.

`npm outdated` output:
```bash
michaelp@MacBook-Air-5 icj-project-rig % npm outdated
npm WARN cli npm v9.7.1 does not support Node.js v16.6.1. This version of npm supports the following node versions: `^14.17.0 || ^16.13.0 || >=18.0.0`. You can find the latest version at https://nodejs.org/.
Package  Current  Wanted  Latest  Location           Depended by
sass      1.63.4  1.63.6  1.63.6  node_modules/sass  icj-project-rig
michaelp@MacBook-Air-5 icj-project-rig % 
```

## Checklist before requesting a review
- [x] All features within the repository still work.

```bash
michaelp@MacBook-Air-5 icj-project-rig % gulp                       
`import sass from 'sass'` is deprecated.
Please use `import * as sass from 'sass'` instead.
[22:13:50] Using gulpfile ~/Documents/GitHub/icj-project-rig/gulpfile.js
[22:13:50] Starting 'default'...
[22:13:50] Starting 'clean'...
[22:13:50] Finished 'clean' after 3.77 ms
[22:13:50] Starting 'styles'...
[22:13:53] Finished 'styles' after 2.53 s
[22:13:53] Starting 'copy'...
[22:13:53] Finished 'copy' after 5 ms
[22:13:53] Starting 'lint'...
[22:13:53] Starting 'scripts'...
[22:13:53] Starting 'images'...
[22:13:53] Finished 'lint' after 404 ms
[22:13:53] gulp-imagemin: Minified 0 images
[22:13:53] Finished 'scripts' after 439 ms
[22:13:53] Finished 'images' after 440 ms
[22:13:53] Starting 'nunjucks'...
[22:13:53] Finished 'nunjucks' after 1.6 ms
[22:13:53] Starting 'bake'...
[22:13:53] Finished 'bake' after 1.97 ms
[22:13:53] Finished 'default' after 2.98 s
michaelp@MacBook-Air-5 icj-project-rig % gulp format                
`import sass from 'sass'` is deprecated.
Please use `import * as sass from 'sass'` instead.
[22:13:58] Using gulpfile ~/Documents/GitHub/icj-project-rig/gulpfile.js
[22:13:58] Starting 'format'...
[22:13:58] Finished 'format' after 108 ms
michaelp@MacBook-Air-5 icj-project-rig % 
```

### CODE REVIEW

**Comment by michplunkett** [comment]
Vulnerabilities before and after changes:
```
39 vulnerabilities (3 low, 10 moderate, 26 high)
...
37 vulnerabilities (3 low, 10 moderate, 24 high)
```

File: tasks/clean.js [hunk 1]
```diff
@@ -1,10 +1,8 @@
-import del from 'del';
+import { deleteSync } from 'del';
 import gulp from 'gulp';
 
 function clean(resolve) {
-  del(['docs/*'], {
-    dot: true
-  });
+  deleteSync(['docs/*']);
```
Comment (michplunkett) [inline tasks/clean.js:5]: `del` does not exist with the current version, so I updated it to the corresponding command.

Comment (michplunkett) [reply tasks/clean.js:5]: https://www.npmjs.com/package/del

Comment (jefflinwood) [reply tasks/clean.js:5]: Seems reasonable!

**Review by jefflinwood — APPROVED** [review]
This PR looks good - thanks for doing the updates!

**Review by critmcdonald — APPROVED** [review]
This all seems fine. Thanks.

**Pull request merged**